content/en/docs/devops-user-guide/examples/go-project-pipeline.md

```diff
@@ -80,31 +78,37 @@ With the above credentials ready, you can create a pipeline using an example Jen
      stages {
        stage('docker login') {
          steps{
-           container ('maven') {
+           container ('go') {
              sh 'echo $DOCKERHUB_CREDENTIAL_PSW  | docker login -u $DOCKERHUB_CREDENTIAL_USR --password-stdin'
-               }
-             }  
            }
+         }
+       }
    
        stage('build & push') {
          steps {
-           container ('maven') {
+           container ('go') {
              sh 'git clone https://github.com/yuswift/devops-go-sample.git'
              sh 'cd devops-go-sample && docker build -t $REGISTRY/$DOCKERHUB_USERNAME/$APP_NAME .'
              sh 'docker push $REGISTRY/$DOCKERHUB_USERNAME/$APP_NAME'
-             }
            }
          }
+       }
        stage ('deploy app') {
          steps {
-           container('maven') {
-             kubernetesDeploy(configs: 'devops-go-sample/manifest/deploy.yaml', kubeconfigId: "$KUBECONFIG_CREDENTIAL_ID")
-             }
+           container('go') {
+              withCredentials([
+                kubeconfigFile(
+                  credentialsId: env.KUBECONFIG_CREDENTIAL_ID,
+                  variable: 'KUBECONFIG')
+                ]) {
+                sh 'envsubst < devops-go-sample/manifest/deploy.yaml | kubectl apply -f -'
+              }
            }
          }
-       }
-     }
-   ```
+      }
+    }
+  }
+  ```
 
    {{< notice note >}}
```

Comment:
Please check the format here in the EN document. The format in the corresponding ZH document is correct.

![Xnip2021-11-16_14-49-30](https://user-images.githubusercontent.com/75110798/141935486-8f24cc81-3094-4f9d-a176-d55f00cc3241.png)

Reply:
fixed